go.mod

```diff
@@ -349,10 +349,5 @@ require (
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
 
-replace (
-	// replicating the replace directive on cosmos SDK
-	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-
-	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
-	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
```

Comment:
seems it's not needed anymore as it has been replaced by `github.com/smartcontractkit/grpc-proxy v0.0.0-20240830132753-a7e17fec5ab7`